docs/user/packaging.rst

```diff
@@ -0,0 +1,15 @@
+Packaging
+=========
+
+You can package the whole Fonduer pipeline (parsing, extraction, featurization, and classification) and deploy it to a remote place to serve.
```

Comment:
It would be awesome to have more content about how to use it and when is a good time to use it? Just to share some idea there or maybe another tutorial for this?

Reply:
Thanks you for your feedback.
I'll add more details to the docs, also I'll update one of the existing tutorials instead of adding another one.

Reply:
Added much more details in the doc.

Despite what I said earlier, I would not update an existing tutorial because it'd require separating some portion of a notebook into a module (like subclasses.py), which may distort what the original tutorial tries to convey.
Also creating another, new tutorial would be good, but requires much more effort.
I might create one in the future, but let me stop here.